packages/loot-design/src/components/AccountAutocomplete.js

```diff
@@ -66,15 +66,23 @@ export function AccountList({
   );
 }
 
-export default function AccountAutocomplete({ embedded, ...props }) {
+export default function AccountAutocomplete({
+  embedded,
+  includeClosedAccounts = false,
```

Comment:
suggestion: would it make more sense to do it the other way round? By defaulting to ignore closed accounts we can introduce subtle bugs in existing functionality and/or future usages of the component. It's very easy to forget about this flag when using the component and thus introduce a bug.

My suggestion would be to default it to `true` (as before) and then manually set it to `false` for some specific edge-cases.

```suggestion
  includeClosedAccounts = true,
```